api/src/main/resources/messages_fr.properties

```diff
@@ -308,6 +308,11 @@ Program.group=Groupe
 Program.agent=Accompagnateur
 Program.states=Statuts
 
+Provider.manage=Prestataire en Charge
+Provider.create=Créer un prestataire
```

Comment:
Are these changes also part of the ticket?

Reply:
The commits weren't updated from master, because new commits had been pushed since the creation of this pull request. I updated the branch and squashed the commits.

Reply:
You still have many un needed changes.
It may be easier to just do it again from a new branch